commit ca4f1d221c84fe364517b15af65f3f0e4ce9719a
Author: Andrew Donnellan <ajd@linux.ibm.com>
Date:   Fri Feb 10 19:03:56 2023 +1100

    powerpc/pseries: Clarify warning when PLPKS password already set
    
    When the H_PKS_GEN_PASSWORD hcall returns H_IN_USE, operations that require
    authentication (i.e. anything other than reading a world-readable variable)
    will not work.
    
    The current error message doesn't explain this clearly enough. Reword it
    to emphasise that authenticated operations will fail.
    
    Signed-off-by: Andrew Donnellan <ajd@linux.ibm.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Link: https://lore.kernel.org/r/20230210080401.345462-22-ajd@linux.ibm.com

diff --git a/arch/powerpc/platforms/pseries/plpks.c b/arch/powerpc/platforms/pseries/plpks.c
index 926b6a927326..01ae919b4497 100644
--- a/arch/powerpc/platforms/pseries/plpks.c
+++ b/arch/powerpc/platforms/pseries/plpks.c
@@ -146,7 +146,7 @@ static int plpks_gen_password(void)
 		memcpy(ospassword, password, ospasswordlength);
 	} else {
 		if (rc == H_IN_USE) {
-			pr_warn("Password is already set for POWER LPAR Platform KeyStore\n");
+			pr_warn("Password already set - authenticated operations will fail\n");
 			rc = 0;
 		} else {
 			goto out;